orcid-integration-test/src/test/java/org/orcid/integration/blackbox/api/v2/rc4/MemberV2ApiClientImpl.java

```diff
@@ -310,7 +311,12 @@ public ClientResponse viewBiography(String orcid, String accessToken) {
         URI uri = UriBuilder.fromPath(BIOGRAPHY).build(orcid);
         return orcidClientHelper.getClientResponseWithToken(uri, VND_ORCID_XML, accessToken);        
     }
-    
+
+    public ClientResponse viewBiographyJson(String orcid, String accessToken) {
+        URI uri = UriBuilder.fromPath(BIOGRAPHY).build(orcid);
+        return orcidClientHelper.getClientResponseWithToken(uri, MediaType.APPLICATION_JSON, accessToken);
+    }
+
     public ClientResponse createKeyword(String orcid, Keyword keyword, String accessToken) {
         URI uri = UriBuilder.fromPath(KEYWORDS).build(orcid);
         return orcidClientHelper.postClientResponseWithToken(uri, VND_ORCID_XML, keyword, accessToken);      
```

Comment:
Hey @wjrsimpson, why you have "viewPersonJson" in the public api but not in the members API?